linetools/isgm/abscomponent.py

```diff
@@ -152,6 +155,8 @@ def __init__(self, radec, Zion, z, vlim, Ej=0./u.cm, A=None, Ntup=None, comment=
             sig_logN : Error in log10 N
         Ej : Quantity, optional
             Energy of lower level (1/cm)
+        stars : str, optional
+            asterisks to add to name, e.g. '**' for CI**
```

Comment:
It seems that `stars` is only relevant when `name=None`. Please add this to the docstring.